umbraco/Umbraco-CMS
# Updated API version package and moved attribute to each controller

### Summary
Changed to use [Asp.Versioning.Http](https://www.nuget.org/packages/Asp.Versioning.Http).

At the same time updated all controllers to have the ApiVersionAttribute on the controller it selves, as this cannot be inherited. (It couldn't before either).

### Test
- Ensure all Apis have same versioning.
- Try to create a v2 endpoint
   - Add `[ApiVersion("2.0")]` to a controller
   - Add `[MapToApiVersion("2.0")]` on a new action.
   - Ensure only this version have v2 and not any other endpoints
